lxd/instance/drivers/driver_qemu.go

```diff
@@ -7159,7 +7159,10 @@ func (d *qemu) diskState() (map[string]api.InstanceStateDisk, error) {
 	}
 
 	disk := map[string]api.InstanceStateDisk{}
-	disk[rootDiskName] = api.InstanceStateDisk{Usage: usage}
+	disk[rootDiskName] = api.InstanceStateDisk{
+		Usage: usage.Used,
+		Total: usage.Total,
+	}
```

Comment:
@stgraber this is causing static analysis failure

Reply:
Oops, fixed now!